datamodel/src/main/java/com/hartwig/serve/datamodel/EfficacyEvidence.java

```diff
@@ -0,0 +1,47 @@
+package com.hartwig.serve.datamodel;
+
+import java.util.Set;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+
+import org.immutables.value.Value;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableEfficacyEvidence.class)
+@JsonDeserialize(as = ImmutableEfficacyEvidence.class)
+@Value.Style(passAnnotations = { NotNull.class, Nullable.class },
+             jdkOnly = true)
+public abstract class EfficacyEvidence {
+
+    @NotNull
+    public abstract Knowledgebase source();
+
+    @NotNull
+    public abstract Treatment treatment();
+
+    @NotNull
+    public abstract String efficacyDescription();
+
+    @NotNull
+    public abstract EvidenceLevel evidenceLevel();
+
+    @NotNull
+    public abstract EvidenceLevelDetails evidenceLevelDetails();
+
+    @NotNull
+    public abstract EvidenceDirection evidenceDirection();
+
+    public abstract int evidenceYear();
+
+    @NotNull
+    public abstract Set<String> evidenceUrls();
```

Comment:
Maybe rename to just `urls` similar to how the urls are called in clinical trial?

This variable used to have `efficacy` in there I believe because there were also other types of urls in the datamodel before (if I recall correctly)